Test/Test.cs

```diff
@@ -1316,6 +1316,33 @@ public void TestLoginEvent()
 
             Assert.Equal("https://api.sift.com/v205/events?abuse_types=legacy,payment_abuse&return_score=true",
                           Uri.UnescapeDataString(eventRequest.Request.RequestUri.ToString()));
+        } 
+
+        [Fact]
+        public void TestScorePercentile()
+        {
+            var sessionId = "sessionId";
+            var transaction = new Transaction
+            {
+                user_id = "vineethk@exalture.com",
+                amount = 1000000000000L,
+                currency_code = "USD",
+                session_id = sessionId,
+                transaction_type = "$sale",
+                transaction_status = "$failure",
+                decline_category = "$invalid"
+            };
+
+            EventRequest eventRequest = new EventRequest
+            {
+                Event = transaction,
+                AbuseTypes = { "legacy", "payment_abuse" },
+                Fields = { "SCORE_PERCENTILES" },

```

Comment:
this if kind of revealing the implementation details. No need for the customer to specify GET-params, this is what happening under the hood. Would be better to provide something like `IncludeScorePercentiles = TRUE` or something like this. Good example is from sift-java - https://github.com/SiftScience/sift-java/commit/6e2781c9f9c80d1bf5b373869be24ecc5b6983f1#diff-9582a6fb9ef5c3c54f41686a60ccead81902999fed7b92f0de1e4da753630b9cR96

Reply:
Done with the said changes

Reply:
Resolved